Library/Formula/camlp5.rb

```diff
@@ -13,20 +13,33 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-latex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on :tex if build.with? "latex"
+  depends_on "gnu-sed" if build.with?("doc") || build.with?("latex")
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
-    end
+    args = ["--prefix", prefix, "--mandir", man]
+    args << "--transitional" if build.without? "strict"
 
-    system "./configure", "-prefix", prefix, "-mandir", man, strictness
-    # this build fails if jobs are parallelized
-    ENV.deparallelize
+    system "./configure", *args
     system "make", "world.opt"
     system "make", "install"
+
+    return if build.without?("doc") && build.without?("latex")
```

Comment:
Is this appropriate, or should I let the remainder of the file do nothing, or wrap the rest in an `if`?